java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -0,0 +1,330 @@
+            	}
+            }
+        });
+		
+		Button fileUploadBtn = (Button) findViewById(R.id.fileUpload);
+		fileUploadBtn.setOnClickListener(new View.OnClickListener() {
+            @SuppressWarnings("unchecked")
+			public void onClick(View view) {
+            	if (service.isAppConnected()) {
+            		String filename = writeFile();
+            		
+            		InputStream source;
+					try {
+						source = openFileInput(filename);
+						
+						File hvFile = new File();
+						hvFile.setName(filename);
+						
+						hvClient.start();
+	            		
+	            		hvClient.asyncRequest(hvFile.uploadAsync(((HVApp)getApplication()).getCurrentRecord(), null, source), 
+	            				new MainActivityCallback(MainActivityCallback.UpdateRecords));
+					} catch (Exception e) {
+						// TODO Auto-generated catch block
+						e.printStackTrace();
+					}
+            	}
+            }
+        });
+		
+		Button vocabTest = (Button) findViewById(R.id.vocabTest);
+		vocabTest.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View view) {
+            	Intent intent  = new Intent(MainActivity.this, VocabActivity.class);
+            	startActivity(intent);
+            }
+        });
+	}*/
+
+	private String writeFile() {
+		String filename = "writefile" + (int)(Math.random() * 100)  + ".txt";
+		String string = "This is from file upload";
+		FileOutputStream outputStream;
+
+		try {
+		  outputStream = openFileOutput(filename, Context.MODE_PRIVATE);
+		  outputStream.write(string.getBytes());
+		  outputStream.close();
+		  
+		  return filename;
+		} catch (Exception e) {
+		  e.printStackTrace();
+		}
+		
+		return null;
+	}
+	
+	private class HVConnect extends AsyncTask<Void, Void, Void> {
+		
+		private ProgressDialog progressDialog;
+        
+        public HVConnect() {
+            progressDialog = ProgressDialog.show(
+                    MainActivity.this,
+                    "",
+                    "Please wait...", 
+                    true);
+        }
+
+		@Override
+		protected Void doInBackground(Void... arg0) {
+			
+			HealthVaultSettings settings = service.getSettings();
+			settings.setMasterAppId("c6ba979f-c342-4408-a2bc-0dfb43b2bf8d");
+			settings.setServiceUrl("https://platform.healthvault-ppe.com/platform/wildcat.ashx");
+			settings.setShellUrl("https://account.healthvault-ppe.com");
+			settings.setIsMultiInstanceAware(true);
+			
+			service.start(MainActivity.this, MainActivity.this);
+			
+			return null;
+		}
+		
+		@Override
+        protected void onPostExecute(Void result) {
+            progressDialog.dismiss();
+        }
+	}
+	
+	public class MainActivityCallback<Object> implements RequestCallback {
+    	public final static int UpdateRecords = 0;
+    
+    	private int event;
+    	
+    	public MainActivityCallback(int event) {
+    		MainActivity.this.setProgressBarIndeterminateVisibility(true);
+    		this.event = event;
```

Comment:
Here we are using "this." to reference member variables, elsewhere we use 'm' prefix, and other places we don't do either. Need to unify these practices.